modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/util/CompanyCountriesUtil.java

```diff
@@ -224,6 +229,35 @@ public static void processCountryRegions(
 		}
 	}
 
+	public static void updateRegionCounter(Connection connection)
+		throws Exception {
+
+		try (Statement statement = connection.createStatement();
+			ResultSet resultSet = statement.executeQuery(
+				"select max(regionId) from Region")) {
+
+			if (resultSet.next()) {
+				CounterLocalServiceUtil.increment(
+					Region.class.getName(), (int)resultSet.getLong(1));
+			}
+		}
+	}
+
+	public static void updateRegionLocalizationCounter(Connection connection)
+		throws Exception {
+
+		try (Statement statement = connection.createStatement();
+			ResultSet resultSet = statement.executeQuery(
+				"select max(regionLocalizationId) from RegionLocalization")) {
+
+			if (resultSet.next()) {
+				CounterLocalServiceUtil.increment(
```

Comment:
If you really need the Counter service, you must use an OSGi reference so the upgrade waits until is ready